src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Utilities/Checksum.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+#if NETCOREAPP
+using System.Runtime.CompilerServices;
+#endif
```

Comment:
@davidwengier: Just wanted to note that this is now necessary due to https://github.com/dotnet/razor/pull/9014. Not sure if you expected that for multi-targeted projects.

Reply:
I hit something similar in that PR, but decided to push on since it was only one case. If it becomes more annoying to you, I have no objection to relaxing the rules again.

Reply:
Why is this necessary? Is one TFM including a global using the other isn't?

Reply:
This is only used for `MemoryMarshal` in the `#if NETCOREAPP` code that below, so it's unneeded on other TFMs.

Reply:
Ah, gotcha.